scripts/ANALYSIS.anasum_parallel_from_runlist.sh

```diff
@@ -290,15 +290,15 @@ for RUN in ${RUNS[@]}; do
         echo "* $RUN $RUN 0 $CUTFILE $BM $EFFAREARUN $BMPARAMS $RADACCRUN"
         if [[ V2DL3 -eq "1" ]]; then
             # write line to v2dl3 script
-            echo "v2dl3 -ed --full-enclosure -f $ODIR/$RUN.anasum.root $VERITAS_EVNDISP_AUX_DIR/EffectiveAreas/$EFFAREARUN $ODIR/$RUN.anasum.fits" >> $V2DL3SCRIPT
+            echo "python  \$PWD/pyV2DL3/script/v2dl3_for_Eventdisplay.py -f $ODIR/$RUN.anasum.root $VERITAS_EVNDISP_AUX_DIR/EffectiveAreas/$EFFAREARUN $ODIR/$RUN.anasum.fits" >> $V2DL3SCRIPT
```

Comment:
Does that mean one has to run the script from the the V2DL3 directory? This is not nice, need to find a better solution.

Reply:
No! One just has to define `PWD` pointing to V2DL3 directory and then it can be run from any directory.

Reply:
You mean as an environmental variable? Use `PWD` as variable name is a bit confusing, as there is a unix command called `pwd` (print working directory) with a different meaning.